src/FakeItEasy/Core/FakeManager.PropertyBehaviorRule.cs

```diff
@@ -39,25 +39,18 @@ public static bool IsPropertyGetter(MethodInfo method)
                 return method.IsSpecialName && method.Name.StartsWith("get_", StringComparison.Ordinal);
             }
 
+            public bool IsMatchForSetter(IFakeObjectCall fakeObjectCall) => IsPropertySetter(fakeObjectCall);

```

Comment:
I guess `IsMatchForSetter` makes it sound to me like we're comparing the rule against a particular setter, but perhaps that's just my brain making assumptions.
This is not such a big deal to me. I appreciate the the explanation.